components/contentblocks/CategoryListBlock.js

```diff
@@ -23,38 +23,12 @@ const CategoryListBlock = (props) => {
   const placemarkerImageUrl = "https://kausal.watch.kausal.tech/media/images/Visit_Tampere_Foggy_sunset_Panorama_from_Nasinn.original.jpg";
   const fakeprops = {
     heading: 'Toimenpidekokonaisuudet',
-    categories: [
-      {
-        id: '1',
-        imageUrl: placemarkerImageUrl,
-        title: 'Toimenpidekokonaisuus 1',
-        progress: '60',
-      },
-      {
-        id: '2',
-        imageUrl: placemarkerImageUrl,
-        title: 'Toimenpidekokonaisuus 2',
-        progress: '60',
-      },
-      {
-        id: '3',
-        imageUrl: placemarkerImageUrl,
-        title: 'Toimenpidekokonaisuus 3',
-        progress: '60',
-      },
-      {
-        id: '4',
-        imageUrl: placemarkerImageUrl,
-        title: 'Toimenpidekokonaisuus 4',
-        progress: '60',
-      },
-      {
-        id: '5',
-        imageUrl: placemarkerImageUrl,
-        title: 'Toimenpidekokonaisuus 5',
-        progress: '60',
-      },
-    ],
+    categories: props.categories.map(({ id, name, imageUrl }) => ({
```

Comment:
This processing is probably unnecessary. We can just change the `{ cat.title }` into `{ cat.name }` below. And we can drop the `progress` for now.